### ISSUE TITLE
Pre-train Log

### ISSUE THREAD

**Issue by choillee82** [mims-harvard/UniTS#37]
First of all, thank you for sharing your excellent research results. 

I am conducting a replication experiment based on your research findings. 

The performance during the pre-training replication process is lower than expected, so I am reaching out to you. 

Could you please share the "**pre-train log of the x128 model**" you trained? 

It would greatly help us identify any mistakes we might have made during the replication process.

Thank you.

**Comment by gasvn** [comment]
Hi, I tried to find the original log and found this. However, since it’s been more than a year since the training, I can’t be certain this is the exact log for the released checkpoint. That said, the trend of the loss should be similar, as there might have only been a few adjustments to the model and training parameters.

[output.log](https://github.com/user-attachments/files/19477803/output.log)

**Comment by choillee82** [comment]
Thank you for sharing the log.
